lib/panels/Dataset/index.js

```diff
@@ -62,8 +62,8 @@ export default React.createClass({
                              onReset={ this.onReset }
                              needsApply={ this.state.opacity !== this.oldOpacity }
                              enabledDefault>
-                  <div className='DatasetPanelContents'>
-                    <i className='fa fa-sun-o' />
+                  <div className={style.contents}>
+                    <i className={style.sun} />
```

Comment:
Same here, naming is important... 

I don't know what is the goal of the sun here. Can I replace it with the moon?